reconcile/utils/oc.py

```diff
@@ -529,9 +540,11 @@ def sa_get_token(self, namespace, name):
     def get_api_resources(self):
         # oc api-resources only has name or wide output
         # and we need to get the KIND, which is the last column
-        cmd = ["api-resources", "--no-headers"]
-        results = self._run(cmd).decode("utf-8").split("\n")
-        return [r.split()[-1] for r in results]
+        if not self.api_resources:
```

Comment:
I was initially wondering if there is a scenario where this could lead to stale information somehow. But I came to the conclusion that api resources should rarely change + this is only cached within one reconcile iteration, so it should be good. Not sure if I forgot sth here.